src/helpers/calculationStepFunctions/numericCalculationFunctions.test.js

```diff
@@ -0,0 +1,38 @@
+import { calculateValue } from "./numericCalculationFunctions";
```

Comment:
Would be great if you could add a note to the project wiki or readme about adding the test framework and how to run the tests.